tests/unit_tests/test_collection_name_validation.py

```
@@ -69,9 +132,327 @@ def test_invalid_characters(self):
+            try:
+                with pytest.raises(ValueError, match="Only letters, digits, and underscore") as exc_info:
+                    _validate_collection_name(name)
+                print(f"输出: ✅ 验证通过 - 捕获到 ValueError: {exc_info.value}")
+            except Exception as e:
+                print(f"输出: ❌ 验证失败 - 未捕获到预期的 ValueError，实际异常: {type(e).__name__}: {e}")
+                raise
+
+    def test_effective_max_length_by_db_limit(self):
+        """Test that collection names respect DB table name limit when combined with prefix."""
+        print(f"\n{'=' * 60}")
+        print(f"测试: test_effective_max_length_by_db_limit - 测试基于DB限制的有效最大长度")
```

Ruff (0.14.10)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)